lib/babe/babe.go

```diff
@@ -128,7 +128,7 @@ func NewService(cfg *ServiceConfig) (*Service, error) {
 		rt:                 cfg.Runtime,
 		transactionState:   cfg.TransactionState,
 		slotToProof:        make(map[uint64]*VrfOutputAndProof),
-		pause:              make(chan struct{}),
+		pause:              make(chan struct{}, 100),
```

Comment:
this doesn't need to be buffered and it's simply closed when the service is paused

Reply:
updated.